matlab/+caffe/private/caffe_.cpp

```diff
@@ -478,6 +478,29 @@ static void read_mean(MEX_ARGS) {
   mxFree(mean_proto_file);
 }
 
+// Usage: caffe_('write_mean', mean_mat, mean_proto_file)
+static void write_mean(MEX_ARGS) {
+  mxCHECK(nrhs == 2 && mxIsSingle(prhs[0]) && mxIsChar(prhs[1]),
+      "Usage: caffe_('write_mean', mean_mat, mean_proto_file)");
```

Comment:
Better to be consistent with matlab code in nomination here and use 'mean_data' instead of 'mean_mat'.